todo_app/app.py

```diff
@@ -1,11 +1,23 @@
-from flask import Flask
-
+from flask import Flask, render_template, redirect, request
+ 
 from todo_app.flask_config import Config
-
+from todo_app.data.session_items import get_items 
+from todo_app.data.session_items import add_item
+ 
 app = Flask(__name__)
 app.config.from_object(Config())
-
-
+ 
+ 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', my_items = items)
+ 
+@app.route('/add_new_items', methods=['POST'])
+def add_new_item():
+    new_item_title = request.form.get('Title')
```

Comment:
Really small but important - your field name here is `Title` but the field name in your form is `title` so this doesn't quite capture the right data as it is; I think we can fix with:
```suggestion
    new_item_title = request.form.get('title')
```